gobblin-core/src/main/java/gobblin/publisher/BaseDataPublisher.java

```diff
@@ -134,51 +147,65 @@ public void publishData(Collection<? extends WorkUnitState> states) throws IOExc
     }
   }
 
-  protected void publishData(WorkUnitState workUnitState, int branchId, Set<Path> writerOutputPathsMoved)
-      throws IOException {
+  private void publishData(WorkUnitState state, int branchId, Set<Path> writerOutputPathsMoved) throws IOException {
+    publishData(state, branchId, false, writerOutputPathsMoved);
```

Comment:
Why line 128 uses `true` while this uses `false` for the third argument? What's the deciding factor here?